modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -26,9 +26,15 @@ image::api-analytics-detail-panel.png[]
 +
 . If you want to view API analytics in API Manager, click the *View API in API Manager* link.
 
+[[enhanced-api-metrics]]
 == Enhanced Metrics View for API Analytics Dashboards
```

Comment:
I don't understand this title and how it's related to the archive API.

Reply:
The heading is for enhanced metrics view and not specifically the API. The basic view is also described in this topic. So, in short, it isn't.